Sources/Cocoa/Components/LabelTextView.swift

```diff
@@ -101,6 +105,14 @@ open class LabelTextView: UITextView {
         textAlignment = .left
         resistsSizeChange(axis: .vertical)
         didTapUrl(Self.defaultDidTapUrlHandler)
+
+        accessibilityHolder = VoiceControlWrapper(textView: self)
+        addSubview(accessibilityHolder!)
+    }
+
+    open override func layoutSubviews() {
```

Comment:
Help needed

I am not sure why auto layout is not working on the UITextView instance, manually I can handle the frames but as soon as I try to use SnapKit the frame is calculated incorrectly.